NSubstitute.DbConnection/NSubstitute.DbConnection/DbConnnectionMockExtensions.cs

```diff
@@ -109,7 +109,7 @@ private static IDbCommand CreateMockCommand(DbConnectionWrapper result)
             mockParameters.Contains(Arg.Any<object>()).Throws<NotSupportedException>();
             mockParameters.Contains(Arg.Any<string>()).Returns(ci => parameters.Any(parameter => parameter.ParameterName == (string)ci[0]));
             mockParameters[Arg.Any<int>()].Returns(ci => parameters[(int)ci[0]]);
-            mockParameters[Arg.Any<string>()].Throws<NotSupportedException>();
+            mockParameters[Arg.Any<string>()].Returns(ci => parameters.First(p => p.ParameterName.Equals((string)ci[0])));
```

Comment:
Is this changed covered by any test cases? It looks right to me though

Reply:
There's no test case explicitly covering this, but I only modified this as I needed it for a different test.